### ISSUE TITLE
Topics for next class - vote or propose

### ISSUE THREAD

**Issue by mmagnuski** [mmagnuski/Biology-of-Brain-and-Behavior#11]
## Instructions:

Each of you has to: 
- vote for some topic number already proposed by me or other students 
- and/or propose a specific topic you would like to hear about or refresh/clarify something that was present on the lectures that this seminar accompanies.
#### VOTING

If you are voting for topic number 4 - please write `VOTE 4` in your comment.
You can vote for up to 3 topics. If you want to vote for topics 1 and 3 you would write:

```
VOTE 1, 3
```

This would mean that you would like both topics 1 and 3 to be covered on our last class, but the prioity goes to topic 1. 
#### PROPOSING

If you are proposing a topic you would like to be covered on our last meeting - plese write:

```
PROPOSE num 
topicdescription
```

Where `num` is topic number (for example 6 if numbers 1 to 5 alreadyhave topics assigned to) `topicdescription` is the descripion of the topic (can be short, but you can write longer descriptions too).
## My ideas for topics

PROPOSE 1
philosophical aspects of neuroscience - the question of free will

PROPOSE 2
neurocognitive experiments on consciousness (and some theories)

PROPOSE 3
networks of the brain (this is not the same as todays topic of brain oscillations)

PROPOSE 4
brush up on basics of neuonal communication - action potentials, ion channels, neurotransmitters, synapses

PROPSE 5
Biological determinants of intelligence

PROPSE 6
The visual system - how visual information travels throught the brain

PROPOSE 7
Neurobiology of decision making

PROPOSE 8
Neurobiological mechanisms of short term memory / working memory

**Comment by maryanisabi** [comment]
VOTE 8, 4, 7

**Comment by dkremen** [comment]
VOTE 3, 5, 8

**Comment by Omito** [comment]
Vote 1, 2, 6

**Comment by martakuczynska** [comment]
Vote 2, 3, 7

**Comment by JedrzejNowe** [comment]
Vote 1, 5, 7

**Comment by suYildiz** [comment]
Vote, 1, 2, 7

**Comment by zzhenna** [comment]
vote  3，7，8

**Comment by ecemcoskun** [comment]
vote, 1,2,7

**Comment by uckoc** [comment]
1, 3, 7

**Comment by juliakrakowiak** [comment]
VOTE 5, 6

**Comment by SuperSanko** [comment]
Vote 1

**Comment by AinaraM** [comment]
Vote 1, 5

**Comment by ioannisstam25** [comment]
Vote 7

**Comment by Hale-Bayatli** [comment]
Vote 2,7,8

**Comment by daryna1** [comment]
Vote 1,2,7

**Comment by aherodzinska** [comment]
Vote 1,5

**Comment by kbrkcc** [comment]
Vote 4,5,8

**Comment by GuShengyin** [comment]
Vote 1,8

**Comment by tetyanasochenko** [comment]
Vote 7,4

**Comment by FerhatOzevli** [comment]
Vote 7,8

**Comment by dmursakaeva** [comment]
Vote 6,7

**Comment by Tubakasaboglu** [comment]
Vote 6,7,8

**Comment by salomedvali** [comment]
Vote for 5th

**Comment by Emrullah571** [comment]
Vote 1,7

**Comment by estefaniay** [comment]
1,7

**Comment by janbankowski** [comment]
vote 8, 1

**Comment by mariasoosai** [comment]
vote 2, 3, 7

**Comment by mmagnuski** [comment]
Dear all, I wrote a simple script that summed up all the votes. Because it was simple it may have missed your vote if you did not follow the instructions closely enough. 
If you voted for more than one topics, subsequent topics were given points as following:

```
first topic - 1 point
second topic - 0.75 points
third topic - 0.5 points
```

## Results:

`Topic` :one: won with `10.75` votepoints.
But `topic` :seven:, with `9.5` points, is the runner-up.
Therefore I am going to cover topic 1, but I will try to add some things from topic 7. :smiley: 

See you on Wednesday!

**Comment by daryna1** [comment]
Dear  Mr Mikolaj Magnuski,
I am Daryna Chadiuk, I will absent today because I have some problems.
Sorry for this. Thank you.
Daryna
понедельник, 20 апреля 2015 г. пользователь Mikolaj Magnuski написал:

**Comment by ioannisstam25** [comment]
I am confused, I have already sent you my project do I have to do something else? I do not have recieved yet any feed back :) 

```
 Στις 4:29 μ.μ. Τρίτη, 16 Ιουνίου 2015, ο/η Mikolaj Magnuski <notifications@github.com> έγραψε:
```

 Closed #11.—

**Comment by mmagnuski** [comment]
Ah, yes, you were one of the first that sent the project. I hope to finish all feedbacking by tomorrow. Sorry for the delay. 
I closed a few issues on github but it is not related to project dealine etc.
